inspirehep/modules/authors/rest/stats.py

```diff
@@ -68,7 +68,7 @@ def serialize(self, pid, record, links_factory=None):
                 "citation_count",
                 "control_number",
                 "facet_inspire_doc_type",
-                "facet_inspire_subjects",
+                "facet_inspire_categories",
```

Comment:
Note how this code didn't work at all, as `facet_inspire_categories` is not part of the `_source` we are originally sending to ES, but is the result of a `copy_to`. But let's keep it, since the Author API will see some love in the future.

Reply:
Will create an issue so that we have it somewhere concretely. 

Also, if this change is ok, then we can merge it?